### ISSUE TITLE
No icons in tray (redshift-gtk, volumeicon...)

### ISSUE THREAD

**Issue by sass432** [bbidulock/icewm#344]
Hallo,
Since the new development of icewm, I miss my usual and till this time funktionable icons, configured over the executabe startup-script:
```
#!/bin/bash

sleep 2 &&
spacefm --desktop &

sleep 3 &&
volumeicon &

sleep 4 &&
redshift-gtk &
```
The desktop funktions of spacefm are funktionable, but instead of volumeicon and redshift-gtk-icon there appears empty black squares.
In between I didn't change my voidlinux system-config's or my .xinitrc with the exec-icewm-session-option.

**Comment by gijsbers** [comment]
I just tried volumeicon and redshift-gtk and their icons do show up in my tray and they do work fine.
As far as I can tell from the Void package template for IceWM they configure with an external systray:

  https://github.com/void-linux/void-packages/blob/master/srcpkgs/icewm/template

The default IceWM configure uses an internal systray, which works fine for me for the applications you mention. The external tray is no longer supported by us. I don't understand the reasons they keep on using the external tray as it had problems and is no longer tested by us.

**Comment by sass432** [comment]
Thank you very much for your answer.
I will inform the members of the void-community and then I hope for a solution!
Because IceWM is the best WindowManager I ever used !
I hope it will keep the stable fast, light and simple features!

**Comment by rubyFeedback** [comment]
> Because IceWM is the best WindowManager I ever used !

I started it using because fluxbox used to be my old generic fallback but unfortunately fluxbox is no longer maintained, and has a weird bug on my system (I can not drag the windows; they all are frozen in place. Can be maximized and minimized but not dragged about).

On IceWM this works fine. I am quite happy with it so far, only the menu entries confuse me a bit (have to look at that xdg stuff, never tried before) ... and there are four workspaces which I don't need (I use only one workspace; they always confused me, need to find out how to disable that).

**Comment by bbidulock** [comment]
Yes, too bad.  I used to use fluxbox alot too.  I really liked that window manager (with the exception that it did not support tiling).  I had some really nice placement algorithms and nice focus handling.  Unfortunately, at some point they destroyed proper signal handling and it would either get locked in a loop or crash when started from xinit.  I gave up trying to help fix it and it looks like the bug has doomed the window manager.  Could always fork it here and fix it like I have done with icewm and blackbox in the past.  Saw some recent commits on the https://github.com/fluxbox/fluxbox mirror, but nothing important in the last 4 years or so.  http://www.fluxbox.org/ is still up and online.  What is the "official" position on maintenance, or have the maintainers abdicated?
